src/components/navigation/TheUserMenu.vue

```diff
@@ -28,6 +37,15 @@
   </v-menu>
   <v-list v-else dense style="width: 100%;">
     <v-subheader>User Profile</v-subheader>
+    <v-list-item v-if="isAuth" @click="logout">
+      <v-list-item-title>Logout</v-list-item-title>
+    </v-list-item>
+    <v-list-item v-else @click="doOpen">
+      <v-list-item-title>Login</v-list-item-title>
+      <v-dialog v-model="dialogOpen" @click:outside="dialogOpen = false" fullscreen>
```

Comment:
and here too

Reply:
do we need to prevent the user from closing the dialog on a background click? What do you think?

Reply:
I think it's fine if clicking outside closes the dialog, it's a fairly common behaviour and in that instance I'd rather have a permissive behaviour